assets/scss/layout/_blocks.scss

```diff
@@ -1,14 +1,29 @@
 .block {
   display: inline-block;
   width: 100%;
-  margin-top: $space-lg;
-  margin-bottom: $space-xxl;
+  margin-top: $space-md;
+  margin-bottom: $space-xl;
+
+  @include large-and-up {
+    margin-top: $space-lg;
+    margin-bottom: $space-xxl;
+  }
 }
 
 .block-header {
   margin-top: 0;
+  margin-bottom: $space-lg;
+
+  @include large-and-up {
+    margin-bottom: $space-xxl;
+  }
 }
 
 .block-footer {
+  margin-top: $space-lg;
   margin-bottom: 0;
+
+  @include large-and-up {
+    margin-top: $space-lg;
```

Comment:
I think this rule can be removed... unless it's overriding something I'm not seeing by specificity, it looks like it's the same as the one above. Ignore me if it's intended.

Reply:
Ah yes, you are right. I added this to make the rule more explicit, like on `block-header`. But in this case it's redundant. I'll amend before merge.